app/celery/tasks.py

```diff
@@ -530,3 +531,12 @@ def process_incomplete_job(job_id):
             process_row(row_number, recipient, personalisation, template, job, job.service)
 
     job_complete(job, job.service, template, resumed=True)
+
+
+@notify_celery.task(bind=True, name="process-ses-result", max_retries=12, default_retry_delay=300000)
```

Comment:
Our other task use max_retries = 5 and default_retry_delay = 300. Why is this one different?

Reply:
Ahh I thought it was milliseconds for some reason so I'll update 30000 to 300.
So the other is 25 mins total time which felt like it would survive a blip but not an outage, 12 is an hour which seemed like a reasonable time to get everything back up. If its longer than 60 mins we have bigger issues that delivery receipts. If we are happy with 25 mins I'll make it consistent.